tools/depends/target/openssl/Makefile

```diff
@@ -43,7 +49,11 @@ $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 		sed -E -ie "s|^CFLAGS=-|CFLAGS=$(CFLAGS) -|" "$(PLATFORM)/Makefile"; \
 		sed -ie "s|-isysroot \$$(CROSS_TOP)/SDKs/\$$(CROSS_SDK) ||" "$(PLATFORM)/Makefile"; \
 		sed -ie "s|static volatile sig_atomic_t intr_signal;|static volatile intr_signal;|" "$(PLATFORM)/crypto/ui/ui_openssl.c"; \
+		if test "$(TARGET_PLATFORM)" = "appletvos"; then \
+			sed -i -- 's/define HAVE_FORK 1/define HAVE_FORK 0/' "$(PLATFORM)/apps/speed.c"; \
+		fi; \
 	fi
+
 	sed -ie "s|apps test||" "$(PLATFORM)/Makefile"; \
```

Comment:
Fixing this command will fix the fork error, see https://github.com/xbmc/xbmc/compare/master...Rechi:depends/appletvos. Updating this step was missed in the OpenSSL 1.1.0 bump.

Reply:
Thank you. Modification done.